aziotctl/aziotctl-common/test-files/apply/manual-x509-est-custom/certd.toml

```diff
@@ -1,7 +1,7 @@
 homedir_path = "/var/lib/aziot/certd"
 [cert_issuance.device-id]
 method = "est"
-url = "https://example.org/.well-known/est"
+url = "http://example.org/.well-known/est"
```

Comment:
Please make new tests like `manual-x509-est-custom-http` instead of changing these ones, so that it's clear that what the test is intended for.

Reply:
Addressed this